test/programmatic-usage.spec.ts

```diff
@@ -36,6 +36,30 @@ describe("when running the programmatic usage", () => {
       expectedTypebox
     );
   });
+  test("generated typebox name based on title attribute support spacing", async () => {
+    const dummySchema = `
+    {
+      "title": "My Contract",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        }
+      },
+      "required": ["name"]
+    }
+    `;
+    const expectedTypebox = addCommentThatCodeIsGenerated(`
+    import { Static, Type } from "@sinclair/typebox";
+
+    export type MyContract = Static<typeof MyContract>;
+    export const MyContract = Type.Object({name: Type.String()}, { $id: "MyContract" });
```

Comment:
Once #54 is merged, this should be changed to

```suggestion
    import { Static, Type } from "@sinclair/typebox";

    export const Module = Type.Module({
      MyContract: Type.Object({ name: Type.String() }, { $id: "MyContract" }),
    });
    export const MyContract = Module.Import("MyContract");
    export type MyContract = Static<typeof MyContract>;
```

Reply:
I think this is just a FYI for now and can be resolved. If otherwise, hit me up.